spec/acceptance/listen_spec.rb

```diff
@@ -50,7 +50,6 @@ class { 'haproxy': }
           listening_service => 'app00',
           server_names      => 'test00.example.com',
           ports             => '5556',
-          options           => 'check',
```

Comment:
For RHEL 9, `socat` doesn't work as expected with `nofork` option which lead to return `502` error code. Removing `nofork` make it working as expected but that fails for other OS.
Hence removing check option here so that things will work as expectation.